spec/classes/init_spec.rb

```diff
@@ -11,6 +11,7 @@
 
       if facts[:operatingsystemmajrelease] == '7'
         it { is_expected.to contain_class('katello::pulp') }
+        it { is_expected.to contain_class('katello::qpid_client') }
```

Comment:
This isn't EL7 specific so in the test it shouldn't be conditional.

Reply:
nice catch

Reply:
This looks swapped. I think `katello::qpid` is EL7-only and `katello::qpid_client` is all versions. At least, https://github.com/theforeman/puppet-katello/blob/a0917838fabc8241e3e5e7f678378a2d45cbea54/manifests/init.pp#L174-L178 implies that.

Reply:
Yep :)